app/models/edition.rb

```diff
@@ -659,7 +659,7 @@ def detailed_format
   end
 
   def content_store_document_type
-    PublishingApiPresenters.presenter_for(self).content.fetch(:document_type)
+    display_type_key
```

Comment:
How confident are you that this is always the case? Searching through the presenters module shows examples where document_type is set to display_type_key, and where it's not, but obviously it's not clear cut, as not all of the things in that module are relevant to editions...

Reply:
I'll have a look

Reply:
The only outlier was detailed guides (which for some reason is `detailed_guidance` in whitehall but `detailed_guide` in the publishing API`), so to avoid confusion I've put the code back to leaning on the presenters but without instantiating all of the content just to get at what is mostly a static string on the presenter.

Reply:
Ok, that looks good :+1: